net/auth/src/TAuthenticate.cxx

```diff
@@ -3212,28 +3213,28 @@ Int_t TAuthenticate::SendRSAPublicKey(TSocket *socket, Int_t key)
 Int_t TAuthenticate::ReadRootAuthrc()
 {
    // rootauthrc family
-   char *authrc = 0;
+   const char *authrc;
```

Comment:
```suggestion
   const char *authrc = nullptr;
```
all pointer must be initialized.

Reply:
Then `TString tRootAuthrc = authrc;` is no longer needed and the test `if (authrc)` needs to be updated to use `IsEmpty()`